_introduction.md

```diff
@@ -1,21 +1,21 @@
 # Introduction
 
-Welcome to the Bitreserve API, we're glad to have you! Provided here is all the documentation you need to help you create custom and revolutionary new services powered by the Bitreserve Platform. With your ingenuity, together we can serve the needs of individuals and organizations across the globe and change the financial services ecosystem forever.
+Welcome to the Uphold API, we're glad to have you! Provided here is all the documentation you need to help you create custom and revolutionary new services powered by the Uphold Platform. With your ingenuity, together we can serve the needs of individuals and organizations across the globe and change the financial services ecosystem forever.
 
 ## Client Libraries
 
 The follow libraries are available:
 
 * [PHP SDK](https://github.com/seegno/bitreserve-sdk-php)
-* [Python SDK](https://github.com/byrnereese/bitreserve-python-sdk)
+* [Python SDK](https://github.com/byrnereese/uphold-python-sdk)
 * [Ruby SDK](https://github.com/groupbuddies/bitreserve)
 
-We are actively looking for developers to contribute Bitreserve client libraries for a variety of platforms and languages. If you have written a library, please share a link to the github repository with us at <a href="mailto:developer@bitreserve.org?subject=I want to contribute code">developer@bitreserve.org</a> so that we can promote it here, and collaborate with you on its further development.
+We are actively looking for developers to contribute Uphold client libraries for a variety of platforms and languages. If you have written a library, please share a link to the github repository with us at <a href="mailto:developer@uphold.com?subject=I want to contribute code">developer@uphold.com</a> so that we can promote it here, and collaborate with you on its further development.
 
 ## API Changes
 
-You can be notified of the latest changes by *watching* our official [documentation repository](https://github.com/bitreserve/api/) on GitHub.
+You can be notified of the latest changes by *watching* our official [documentation repository](https://github.com/uphold/api/) on GitHub.
```

Comment:
This should be done in another PR since the organization rename hasn't been made yet.